astropy/time/utils.py

```diff
@@ -60,14 +60,16 @@ def day_frac(val1, val2, factor=None, divisor=None):
 
     # get integer fraction
     day = np.round(sum12)
-    extra, frac = two_sum(sum12, -day)
-    frac += extra + err12
-    # Our fraction can now have gotten >0.5 or <-0.5, which means we would
-    # loose one bit of precision. So, correct for that.
-    excess = np.round(frac)
+    # Calculate remaining fraction. This can have gotten >0.5 or <-0.5, which means
```

Comment:
@mhvk , given that this is a legit bug fix, do we need a change log?

Reply:
I don't think so, the exact details of how `day_frac` works are really an implementation detail -- we've never promised `Time` would keep the last bit of precision.